### ISSUE TITLE
Old rundeck version and CVE-2021-44228

### ISSUE THREAD

**Issue by pendharker** [rundeck/rundeck#7431]
_Note: Bug Reports not following this template may be closed without feedback._

**Getting Help**
Please use this issue template for reporting identified or suspected **bugs** *only*. For support on configuration or troubleshooting please see [Getting Help](https://docs.rundeck.com/docs/manual/02-getting-help.html#enterprise-support) for the appropriate channels.

**Describe the bug**

**My Rundeck detail**

* Rundeck version: [ 2.6.7]
* install type: [choose: deb]
* OS Name/version: [Debian]
* DB Type/version: [mysql ]

**To Reproduce**
1. Go to 'CVE-2021-44228'

**Expected behavior**
CVE-2021-44228

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [Windows]
 - Browser [Chrome]
 - Version [latest]

**Additional context**

I am running multiple old versions of Rundeck (2.6.7 and 2.11.5) and need to know how to address this latest log4j vulnerability. I tried the suggestions on other similar issue reported in this Github space but no luck. Can someone suggest the resolution?

**Comment by 12Solved** [comment]
AFIAK older versions of rundeck use logback, which is not vulnerable to log4shell.

For example:
```
$ git checkout v3.2.1
$ grep log4j gradle.properties build.gradle
$ grep logback gradle.properties build.gradle gradle.properties
35:logback.version=1.2.3
find . -name "log4j2*"
```

**Comment by zorroDes** [comment]
Hi @12Solved and for newer version Is there something scheduled to upgrade log4j to version 2.15.0 ?

**Comment by MegaDrive68k** [comment]
Hi @zorroDes  The fix is available for [3.4](https://github.com/rundeck/rundeck/pull/7427) and for [3.3](https://github.com/rundeck/rundeck/pull/7429) will be ready soon. Regards!

**Comment by zorroDes** [comment]
Thanks for the quick, very much appreciated !

**Comment by fdevans** [comment]
HI @zorroDes,
  @MegaDrive68k is correct.  the 3.3.15 version will be available in the next day or two and 3.4.7 is already released.  We won't be going backwards any farther than those versions.  So much new stuff in the latest versions it's strongly encouraged to upgrade.
